ITKImageProcessingFilters/ITKStitchMontage.cpp

```diff
@@ -160,16 +159,46 @@
   }                                                                                                                                                                                                    \
   else                                                                                                                                                                                                 \
   {                                                                                                                                                                                                    \
-    filter->setErrorCondition(TemplateHelpers::Errors::UnsupportedDataType, "The input array's data type is not supported");                                                 \
+    filter->setErrorCondition(TemplateHelpers::Errors::UnsupportedDataType, "The input array's data type is not supported");                                                                           \
   }
 
 #define EXECUTE_STITCH_FUNCTION_TEMPLATE(filter, call, inputData, ...) EXECUTE_DATATYPE_FUNCTION_TEMPLATE(filter, call, inputData, __VA_ARGS__)
 
+namespace
+{
+QString generateDataContainerName(const QString& dataContainerPrefix, const IntVec2Type& montageStart, const IntVec2Type& montageEnd, int32_t row, int32_t col)
+{
+  IntVec3Type montageSize;
+  std::transform(montageStart.begin(), montageStart.end(), montageEnd.begin(), montageSize.begin(), [](int32_t a, int32_t b) -> int32_t { return a + b + 1; });
```

Comment:
What is this line supposed to be doing?

Reply:
I assume that it computes the size of the montage and stores it in montageSize, using the montageStart and montageEnd?  I grabbed that code from ITKPCMTileRegistration, which it looks like you wrote on 5/23/19.